geopyspark-backend/project/Version.scala

```diff
@@ -1,7 +1,7 @@
 object Version {
   val geopyspark = Environment.sha.getOrElse("0.4.2")
   val geotrellis = "3.0.0-SNAPSHOT"
```

Comment:
Should use `2.2.0` now that it is released, there is corresponding version of geotrellis-contrib-vlm that is available.